app/notebook-store.js

```diff
@@ -186,5 +200,6 @@ module.exports = {
      */
     get: get,
     remove: remove,
-    upload: upload
+    upload: upload,
+    getNotebooks: getNotebooks
```

Comment:
Having both `get` and `getNotebooks` is ambiguous. Keeping with the nomenclature, we could use `list` here.